src/components/header/modals/ModalConnectWallet.vue

```diff
@@ -186,6 +207,9 @@ export default defineComponent({
     const { currentAccountName, disconnectAccount } = useAccount();
     const isClosing = ref<boolean>(false);
     const { currentNetworkIdx } = useNetworkInfo();
+    const isAccountUnification = computed<boolean>(() => {
+      return !!(currentNetworkIdx.value === endpointKey.SHIBUYA && currentAccountName);
```

Comment:
A small suggestion. Maybe to enable account unification UI on a local node also

Reply:
@bobo-k2 　I've fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/935/commits/522c5c6a120ca98ffdcaeabee7fe633044250c61) commit